spec/models/concerns/notable_and_taggable.rb

```diff
@@ -4,12 +4,20 @@
   describe '#all_notes' do
     subject { record.all_notes }
 
-    before { record.tag_string = 'foo' }
+    before { record.tag_string = 'foo:1' }
```

Comment:
might be worth making this an additional spec so that we're testing both `foo` and `foo:1`?

Reply:
Yep good shout, I have added this in a fixup.

Before this PR we had:
- PublicBody tagged with "foo:bar" would be associated with a note with tagged with "foo:bar" but not "foo"
- PublicBody tagged with "foo" would be associated with a note with tagged with "foo" but not "foo:bar"

After this PR:
- PublicBody tagged with "foo:bar" would be associated with a note with tagged with "foo:bar" **or** "foo"
- PublicBody tagged with "foo" would be associated with a note with tagged with "foo" but not "foo:bar"


What do you think about that last case... should a PublicBody tagged with "foo" be associated with a note with tagged with "foo:bar".

Reply:
> After this PR:
>
> * PublicBody tagged with "foo:bar" would be associated with a note with tagged with "foo:bar" or "foo"
> * PublicBody tagged with "foo" would be associated with a note with tagged with "foo" but not "foo:bar"

This sounds like the correct behaviour to me.